lxd/api_internal.go

```diff
@@ -789,11 +789,11 @@ func internalImportFromBackup(d *Daemon, projectName string, instName string, fo
 		return err
 	}
 
-	// Generate a new UUID for the instance doing a backup.
-	// We will later use this new instance UUID as a VM Generation ID
-	// in order to differentiate the two instances before and after snapshot
-	// restore that changes that instance's place in time (moving it backwards)
-	instDBArgs.Config["volatile.uuid"] = uuid.New()
+	// If the instance is a VM, generate a new `volatile.uuid.vmggenid`
```

Comment:
Is there an extra "g" in that `vmggenid`? If yes, this was copy-n-pasted to other places.